src/lsst/cmservice/handlers/functions.py

```diff
@@ -132,30 +154,43 @@ async def create_script_template(
     config_values: dict
         Values for the ScriptTemplate
 
+    allow_update: bool
+        Allow updating existing templates
+
     Returns
     -------
     script_template: ScriptTemplate
-        Newly created ScriptTemplate
+        Newly created or updated ScriptTemplate
     """
     key = config_values.pop("name")
     script_template_q = select(ScriptTemplate).where(ScriptTemplate.fullname == key)
     script_template_result = await session.scalars(script_template_q)
     script_template = script_template_result.first()
-    if script_template:
+    if script_template and not allow_update:
         print(f"ScriptTemplate {key} already defined, skipping it")
         return None
-    return await ScriptTemplate.load(
-        session,
-        name=key,
-        file_path=config_values["file_path"],
-    )
+    if script_template is None:
+        return await ScriptTemplate.load(
+            session,
+            name=key,
+            file_path=config_values["file_path"],
+        )
+    full_file_path = os.path.abspath(os.path.expandvars(config_values["file_path"]))
```

Comment:
Asymmetry with file handling here, vs. the load case.  Is there a good reason?

Reply:
🤓

Reply:
I was lazy & didn't want to right a new function.